public/react-services/filter-authorization-agents.ts

```diff
@@ -0,0 +1,52 @@
+/*
+* Wazuh app - React component for get authorized agents.
+* Copyright (C) 2015-2021 Wazuh, Inc.
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* Find more information about this on the LICENSE file.
+*/
+import { AppState } from '../react-services/app-state';
+
+ 
+ 
+export function getFilterWithAuthorizedAgents(agentsIds) {
+  if(!agentsIds)
+    return ;
+   //check for empty agents array
+   if(agentsIds.length == 0){return }
```

Comment:
You could simplify the empty `return` with 2 conditions using the `||` operator:
```
if(!agentsIds || agentsIds.length === 0){ return };
```

Reply:
I've already changed it, thanks for the tip!